lib/apt_news.py

```diff
@@ -22,17 +24,18 @@ def main(cfg: UAConfig):
 
 
 if __name__ == "__main__":
+    root_logger.propagate = False
```

Comment:
I don't think we need to set this as long as we don't configure the root logger (the actual python root logger), right?

Reply:
Yes we do not need to set this property. After testing I noticed my worries are not valid, since other teams/people will be using pro via the api endpoints.